### ISSUE TITLE
Create new XML file

### ISSUE THREAD

**Issue by sevendials** [hercules-team/augeas#536]
It doesn't seem possible to create a new XML file as is possible with other files. When I try, I get this:
```
set /augeas/load/Shellvars/incl[0] /tmp/test.xml
load
set /files/tmp/test.xml/test #empty
save
error: Failed to execute command
saving failed (run 'errors' for details)
errors
...
Error in /tmp/test.xml:2.0 (parse_skel_failed)
  Syntax error
  Lens: /opt/puppetlabs/puppet/share/augeas/lenses/dist/xml.aug:153.10-155.60:
```

**Comment by lutter** [comment]
The problem is that you are trying ot use the `Shellvars` lens for an XML file with your first line. This works:
```
set /augeas/load/Xml/incl[last()+1] /tmp/test.xml
load
set /files/tmp/test.xml/test #empty
save
```

That gives me a `/tmp/test.xml` with `<test/>` in it

**Comment by lutter** [comment]
And, just for extra geek points as a one-liner:

```
augtool -s -A -t 'Xml.lns incl /tmp/test.xml' 'set /files/tmp/test.xml/test #empty'
```

**Comment by sevendials** [comment]
Thanks for the quick response and the nifty one-liner.

My copy/paste had a typo and the problem really does happen in Augeas 1.4 but it is fixed by using the lens from 1.9 so I can use that, cheers!
